ws-service/types.ts

```diff
@@ -1,8 +1,21 @@
+import { Prisma } from '@prisma/client'
 import { TransactionWithAddressAndPrices } from 'services/transactionService'
 
-type TxBroadcast = 'NewTx' | 'OldTx'
+type TxBroadcastType = 'NewTx' | 'OldTx'
+
 export interface BroadcastTxData {
   address: string
-  txs: TransactionWithAddressAndPrices[]
-  messageType: TxBroadcast
+  txs: Array<TransactionWithAddressAndPrices | SimplifiedTransaction>
```

Comment:
this is no good, makes so that `txs` could be an array containing both types, which will never happen.

furthermore, we should either remove the possiblity of having `TransactionWithAddressAndPrices` here and parse it into the simplified version before broadcasting from the nextJS server to the websocket express server (check `services/chronikService.ts:270` for an example of where this is done) OR we should do this parsing before broadcasting from the websocket express server to the clients connected. First option is better IMO, removes the need for another type and saves memory & bandwith (the first advantage is the most relevant)

Reply:
I see, I will update the `txs` in `BroadcastTxData` then so all broadcasted transactions are simplified